### DISCUSSION TITLE
Percentile Window Function

### DISCUSSION THREAD

**Discussion by samfaar** [pawelsalawa/sqlitestudio#4048] [Q&A]
### Details
In many occasions, one needs to calculate a given percentile of a column, e.g. median (50th percentile), or top 5% (95th percentile), but the lack of a window function makes the calculation of this simple statistical parameter complex in SQLite (As opposed to other DBMS'). It would be very helpful if such functionality is adde to SQLite as a window function (similar to Postgres: Percentile(0.95) OVER (...)).

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
Firstly, this is SQLiteStudio, an application that uses SQLite. It's not the SQLite itself.
Secondly, SQLite does support window functions: https://sqlite.org/windowfunctions.html#biwinfunc
